src/main/java/com/easypost/model/TimeInTransit.java

```diff
@@ -1,67 +1,152 @@
+/**
+ * TimeInTransit.java
+ * This file is a part of EasyPost API SDK.
+ * (c) 2022 EasyPost
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
 package com.easypost.model;
 
-public class TimeInTransit {
-    Integer percentile_50;
-    Integer percentile_75;
-    Integer percentile_85;
-    Integer percentile_90;
-    Integer percentile_95;
-    Integer percentile_97;
-    Integer percentile_99;
+public final class TimeInTransit {
 
+    /*
+    Using underscores in a field name violates the style guide,
+    but this is necessary because of how we do JSON deserialization.
+    (We just Gson, which expects a one-to-one match between JSON keys and Java field names.)
+    This can (potentially) be rectified by using something like Jackson in the future to de/serialize
+    via setters and getters, while the field names could follow proper style.
```

Comment:
>     (We just Gson, which expects a one-to-one match between JSON keys and Java field names.)

Nit: one-to-one doesn't mean identical. 

More importantly, neither one-to-one nor identical matching are real expectations for gson, only the default behavior. 

See e.g. `@SerializedName`
https://www.javadoc.io/doc/com.google.code.gson/gson/2.6.2/com/google/gson/annotations/SerializedName.html
and/or
`FieldNamingPolicy` `LOWER_CASE_WITH_UNDERSCORES` https://www.javadoc.io/doc/com.google.code.gson/gson/2.6.2/com/google/gson/FieldNamingPolicy.html

Reply:
Oh, that's a good point. We could name the variables according to the style guide, and then just pop a @SerializedName annotation on them to clarify their JSON key name.

In fact, I think I'd like to do that for all fields.

Reply:
I went ahead and added the annotations in TimeInTransit.java to allow the percentile fields to be de/serialized properly, while also allowing the field names to comply with our style guide (1152995). The getter and setter functions have been renamed, so this is a BREAKING CHANGE.

TimeInTransit was the only place where this was an issue, since all our other JSON fields either are not snake case or are handled by the FieldNamingPolicy we employ in Gson.

That being said, I think it would be smart in the future to explicitly declare the JSON key names by adding SerializedName to all fields. This would help in the future if a JSON key were to change, as we wouldn't have to change the Java field name, just update the annotation.

Reply:
👍  Withdrawing my block.